troubleshooting/known_issues/orphaned_cfs_pods.md

```diff
@@ -0,0 +1,34 @@
+# Orphaned CFS Pods After Booting or Rebooting
+
+After a boot or reboot a few CFS Pods may continue running even after they've
```

Comment:
Change to:

After a boot or reboot, a few CFS pods may continue running after they have finished and may never go away.